bin/juttle

```diff
@@ -214,6 +217,10 @@ function perform_run(options) {
             program.deactivate();
         }).finally(function() {
             current_program = undefined;
+
+            if (had_an_error) {
+                return Promise.reject();
```

Comment:
It's better practice to reject with an `Error` of some kind so the calling code knows how to distinguish this from some other error.